docs/infrastructure.md

```diff
@@ -1,10 +1,11 @@
-# SysAdmin
+These instructions are for a sysadmin looking to 
+deploy a new Neurobagel node locally in an institute or lab. 
+A local neurobagel node includes the neurobagel API and
+a graph backend to store the harmonized metadata.
```

Comment:
That's a good idea. If we do that, I'd deal with it in a different PR though. Otherwise this here turns into "rewrite docs better" and that'll take even longer